### PR TITLE
NetMauMau V0.15

### PR DESCRIPTION
**Pull request by velnias75**
:star2: Live long and prosper :star:

### CODE REVIEW

File: games-board/netmaumau/netmaumau-0.15.ebuild [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+MY_P=nmm-qt-client${PV}
+
+DESCRIPTION="Client for games-server/netmaumau, the popular card game Mau Mau"
+HOMEPAGE="http://sourceforge.net/projects/netmaumau"
+SRC_URI="https://github.com/velnias75/NetMauMau-Qt-Client/archive/V${PV}.tar.gz -> ${P}-client.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="no-espeak"
```
Comment (hasufell) [inline games-board/netmaumau/netmaumau-0.15.ebuild]: I find ```no-foo``` USE flags confusing. What's wrong with ```IUSE="+espeak"```?

Comment (velnias75) [inline games-board/netmaumau/netmaumau-0.15.ebuild]: Once we had the discussion about a plus (+) IUSE flag...

As you know I hate to make new or edit PR's. So please modify as you like as long as to use espeak is the default behaviour.

Comment (hasufell) [inline games-board/netmaumau/netmaumau-0.15.ebuild]: > As you know I hate to make new or edit PR's

Why's that? It's the most common operation after ```git commit```.
You can do it either via an interactive rebase:

```
git rebase -i HEAD~2
# change 'pick' to 'e' at the offending commit
# save editor and exit
# git rewinds to that commit now
# fix the games-board/netmaumau ebuild
git add games-board/netmaumau
git commit --amend
git rebase --continue
```

 or just do a mixed reset and repeat both commits:

```
git reset --mixed HEAD~2
# fix games-board/netmaumau ebuild
git add games-board/netmaumau
git commit -ma '<commit message>'
git add games-server/netmaumau
git commit -ma '<commit message>'
```

Comment (velnias75) [inline games-board/netmaumau/netmaumau-0.15.ebuild]: Force push?

File: games-board/netmaumau/netmaumau-0.15.ebuild [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+MY_P=nmm-qt-client${PV}
+
+DESCRIPTION="Client for games-server/netmaumau, the popular card game Mau Mau"
+HOMEPAGE="http://sourceforge.net/projects/netmaumau"
+SRC_URI="https://github.com/velnias75/NetMauMau-Qt-Client/archive/V${PV}.tar.gz -> ${P}-client.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+espeak"
```
Comment (velnias75) [inline games-board/netmaumau/netmaumau-0.15.ebuild]: Force pushed! :grinning:

**Comment by hasufell** [comment]
had to fix v0.14 too 62fdeef15c8788685b8a0d851a01a1345161db2e

**Pull request merged**